src/utils/ContainerUtils.ts

```diff
@@ -0,0 +1,50 @@
+export type ButtonType = "first" | "next" | "previous" | "last";
```

Comment:
Would personally have put all these props (also `PaginationContainerProps`, `PaginationContainerState`, `ListView`) where they're being used. They're not being used in this file so they don't add value here and make a used have to navigate to another file to figure things out.
`WrapperProps` could perhaps be moved to a global typings file.

Reply:
Before i was using the  list view navigation button before i stopped using them so they were needed then. But now i have mode type to MendixButton component